package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "cypress-training",
+  "version": "1.0.0",
+  "description": "This is a Workshop about Cypress",
+  "main": "index.js",
+  "scripts": {
+    "test": "cypress open"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/opensystemxfr/cypress-training.git"
+  },
+  "keywords": [
+    "ui-testing",
+    "dojo",
+    "practice",
+    "cypress"
+  ],
+  "author": "[Su nombre] <[Su correo]> ([su github])",
```

Comment:
The idea here is to fill out with your data: Name <mail> (github user) example:
`Holmes Salazar <mymail@gmail.com> (holgiosalos)`